bundles/org.openhab.binding.solarforecast/src/main/java/org/openhab/binding/solarforecast/internal/actions/SolarForecastActions.java

```diff
@@ -172,15 +172,15 @@ public void triggerUpdate() {
         }
     }
 
-    public static State getDay(ThingActions actions, LocalDate ld, String... args) {
+    public static QuantityType<Energy> getDay(ThingActions actions, LocalDate ld, String... args) {
```

Comment:
Not related to this PR, but the method name is somewhat confusing. its getting the energy state, not the day/datetime.

Reply:
Ok, method renamed into `getEnergyOfDay`

Reply:
I think this is much better, but be aware this is a breaking change (users have to adapt their rules). And the readme needs to be adapted too, but that is no real problem.

Reply:
README is now updated.

For the breaking change, a PR in distro is required.